dotnetcementrefs/dotnetcementrefs.cs

```diff
@@ -186,12 +186,12 @@ private static bool ShouldIgnore(Project project)
 
 
     private static ProjectItem[] FindCementReferences(Project project, ISet<string> localProjects,
-        string[] cementRefPrefixes)
+        string[] cementRefPrefixes, bool allowLocalProjects)
     {
         return project.Items
             .Where(item => item.ItemType == "Reference")
             .Where(item => cementRefPrefixes.Any(x => item.EvaluatedInclude.StartsWith(x)))
-            .Where(item => !localProjects.Contains(item.EvaluatedInclude))
+            .Where(item => allowLocalProjects || !localProjects.Contains(item.EvaluatedInclude))
```

Comment:
Are you sure, that `allowLocalProjects` should be used inside `FindCementReferences`?
Can we just add them later?

Reply:
Sounds legit. 

I also thought that maybe we should explicitly specify local projects to resolve

Reply:
Something like `--includeLocal ProjectA ProjectB`

Reply:
Seems better to exclude some projects later (in super special cases).

Reply:
And it should be very similar with `ReferencesToRemove`.